chart/templates/deployment.yaml

```diff
@@ -0,0 +1,171 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "aks-mcp.fullname" . }}
+  labels:
+    {{- include "aks-mcp.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
```

Comment:
would it actually work for multiple replicas? I don't think we have distributed login state for oauth yet?